cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -1,61 +1,146 @@
+{% import '_vars-mega-menu.html' as vars with context %}
+{% set base_class = 'o-mega-menu' %}
 
 {# ==========================================================================
 
-   sub_nav()
+   _classes()
 
    ==========================================================================
 
    Description:
 
-   Creates a mega menu sub menu content markup when given:
+   Creates mega menu CSS classes when given:
+
+   nav_depth:    Level of menu nesting. 1 equals the root menu.
+
+   suffix:       Suffix to add to the classes.
+                 Default is ''.
+
+   ========================================================================== #}
+
+{%- macro _classes( nav_depth, suffix='' ) -%}
+{%- set general_class = base_class ~ '_content' -%}
+{%- set depth_class = base_class ~ '_content-' ~ nav_depth -%}
```

Comment:
Any reason not to make this `general_class ~ nav_depth` so that you only have to update one if `_content` changed for some reason?